app/background.cc

```diff
@@ -183,11 +185,17 @@ int grab_background(std::shared_ptr<background_t> pbkd, int width, int height, c
     if (pbkd->video) {
         // grab frame & frame no. under mutex
         std::unique_lock<std::mutex> hold(pbkd->rawmux);
-        cv::resize(pbkd->raw, out, cv::Size(width, height));
+        cv::Rect crop = calcCropping(pbkd->raw.cols, pbkd->raw.rows, width, height);
```

Comment:
We could lift this calculation out of the processing loop to a structure variable, as sizes will not change during operation.

Reply:
see below

Reply:
Size will not change, the frame content will change. Mofifying this implies:
if ! rect_calculated
  crop =  calc_cropping()
else
   crop = get_stored_crop_value()
...
I don't think that the calculation will take a lot ot time more than filling the cv::Rect from already calculated variables.